doc/rados/operations/placement-groups.rst

```diff
@@ -559,100 +559,103 @@ resources.
-that is 512 placement groups per OSD. That does not use too many
-resources. However, if 1,000 pools were created with 512 placement
-groups each, the OSDs will handle ~50,000 placement groups each and it
-would require significantly more resources and time for peering.
+For example, suppose you have a cluster of 10 pools, each with 512 PGs on 10
+OSDs. That amounts to 5,120 PGs distributed across 10 OSDs, or 512 PGs per OSD.
+This cluster will not use too many resources. However, in a cluster of 1,000
+pools, each with 512 PGs on 10 OSDs, the OSDs will have to handle ~50,000 PGs
+each. This cluster will require significantly more resources and significantly
+more time for peering.
 
-You may find the `PGCalc`_ tool helpful.
+For determining the optimal number of PGs per OSD, we recommend the `PGCalc`_
+tool.
 
 
 .. _setting the number of placement groups:
 
-Set the Number of Placement Groups
-==================================
+Setting the Number of PGs
+=========================
 
-To set the number of placement groups in a pool, you must specify the
-number of placement groups at the time you create the pool.
-See `Create a Pool`_ for details.  Even after a pool is created you can also change the number of placement groups with:
+Setting the initial number of PGs in a pool must be done at the time you create
+the pool. See `Create a Pool`_ for details. However, even after a pool is
+created, you can change the number of PGs by running a command of the following
+form:
 
 .. prompt:: bash # 
 
    ceph osd pool set {pool-name} pg_num {pg_num}
 
-After you increase the number of placement groups, you must also
-increase the number of placement groups for placement (``pgp_num``)
-before your cluster will rebalance. The ``pgp_num`` will be the number of
-placement groups that will be considered for placement by the CRUSH
-algorithm. Increasing ``pg_num`` splits the placement groups but data
-will not be migrated to the newer placement groups until placement
-groups for placement, ie. ``pgp_num`` is increased. The ``pgp_num``
-should be equal to the ``pg_num``.  To increase the number of
-placement groups for placement, execute the following:
+If you increase the number of PGs, your cluster will not rebalance until you
+increase the number of "PGs for placement" (``pgp_num``). The ``pgp_num``
+parameter specifies the number of PGs that are to be considered for placement
+by the CRUSH algorithm. Increasing ``pg_num`` splits the PGs in your cluster,
+but data will not be migrated to the newer PGs until ``pgp_num`` is increased.
+The ``pgp_num`` parameter should be equal to the ``pg_num`` parameter. To
+increase the number of PGs for placement, run a command of the following form:
 
 .. prompt:: bash #
 
    ceph osd pool set {pool-name} pgp_num {pgp_num}
 
-When decreasing the number of PGs, ``pgp_num`` is adjusted
-automatically for you.
+If you decrease the number of PGs, then ``pgp_num`` is adjusted automatically.
```

Comment:
When running a Ceph release from Nautilus onwards, and not using the pg_autoscaler, ``pgp_num`` will be automatically stepped to match ``pg_num``.  Note that this process will manifest as periodic periods of remapped PGs and backfill, which is normal and expected.

Reply:
Accepted.